tests/utils/mod.rs

```diff
@@ -1,4 +1,4 @@
-use failure::Error;
+use anyhow::Result;
```

Comment:
Let's maybe avoid importing a type under the name of an existing one? Please either rename it or not import it and always refer to it with `anyhow::Result`.